src/io_uring/tests/integration_tests.rs

```diff
@@ -76,12 +84,92 @@ fn test_eventfd() {
     }
 }
 
+#[test]
+fn test_enable() {
+    skip_if_kernel_lt_5_10!();
+
+    // The ring was not created as disabled.
+    {
+        let ring = IoUring::new(NUM_ENTRIES, true).unwrap();
+        assert!(ring.enabled());
+    }
+
+    // The ring was created as disabled.
+    {
+        let mut ring = IoUring::new(NUM_ENTRIES, false).unwrap();
+        assert!(!ring.enabled());
+        assert!(ring.enable().is_ok());
+        assert!(ring.enabled());
```

Comment:
nit: We could also verify here that enabling an already enabled ring is ok.

Reply:
`enable()` is no longer a public function